test/units/plugins/lookup/test_password.py

```diff
@@ -466,24 +466,26 @@ def test_only_a(self, mock_get_paths, mock_write_file):
         for result in results:
             self.assertEquals(result, u'a' * password.DEFAULT_LENGTH)
 
-    def test_lock_mechanism(self):
+    def test_lock_been_held(self):
         # pretend the lock file is here
         password.os.path.exists = lambda x: True
         try:
             with patch.object(builtins, 'open', mock_open(read_data=b'hunter42 salt=87654321\n')) as m:
                 # should timeout here
                 results = self.password_lookup.run([u'/path/to/somewhere chars=anything'], None)
-                self.assertEquals(True, False)
+                self.fail('Lookup didnt timeout when lock already been held, testcase failed')
```

Comment:
`s/didnt/didn't/`

Reply:
you don't need to say `testcase failed`, it's obvious from testrunner's indication